src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.prog101CoffeeMaker.coffeeMaker.testPieces;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StartButtonTest {
+
+    @Test
+    public void isPressed() {
```

Comment:
You must follow the convention about test names, in all methods

Reply:
Ok boss